css/properties/background-attachment.json

```diff
@@ -80,7 +80,8 @@
                 "version_added": "5"
               },
               "safari_ios": {
-                "version_added": "4.2"
+                "version_added": "4.2",
+                "version_removed": "13"
```

Comment:
This will need to become an array of support statements. Something like:

```
[
  {
    "version_added": "13",
    "partial_implementation": true,
    "notes": "<code>local</code> is recognized but has no effect."
  },
  {
    "version_added": "4.2",
    "version_removed": "13"
  }
]
```